app/data/AtomListStore.scala

```diff
@@ -80,7 +80,16 @@ class CapiBackedAtomListStore(capi: CapiAccess) extends AtomListStore {
         (asset \ "version").as[Long]
       }
 
-      Some(MediaAtomSummary(id, title, posterImage, contentChangeDetails))
+      val currentAsset = (atom \ "assets").as[JsArray].value.find { asset =>
+        val assetVersion = (asset \ "version").as[Long]
+        activeVersion.contains(assetVersion)
```

Comment:
effectively we want to find the asset where activeVersion == assetVersion, but since activeVersion is an `Option[Long]` and assetVersion is just a `Long`, we use `.contains`